hw/ip/rv_core_ibex/doc/boot-rom-patching.md

```diff
@@ -277,37 +290,37 @@ h8944: sub r4, r3, r1
 h8948: add r5, r4, r2
 ```
 
-The instruction `add r2, r3, x0` needs to be replaced with the sequence `add r2, r3, x0; sub r2, r4`.
+The instruction `add r2, r3, x0` needs to be replaced with the sequence `add r2, r3, x0; sub r2, r4, r6`.
 Because the new sequence is more than one instruction we have to redirect the execution flow.
 Let's allocate a SRAM location, say `h0100_0040` and place the patched sequence:
 
 ```
-h100_0040: add r2, r3, x0     // The patched instruction
-           sub r2, r4         // New instruction patched in
-           sub r4, r3, r1     // Orig instruction at h8944
-           auipc r31, disp_h100004c_to_h8948
-           jalr x0, r31
+h1000_0040: add r2, r3, x0     // The patched instruction
+            sub r2, r4, r6     // New instruction patched in
+            sub r4, r3, r1     // Orig instruction at h8944
+            auipc r31, disp_h1000_004c_to_h8948
+            jalr x0, r31
 ```
 
-Let's allocate a trampoline location at say `h100_0000` for this patch.
+Let's allocate a trampoline location at say `h1000_0000` for this patch.
 
 ```
-h100_0000: auipc r31, disp_from_8940_to_100_0040
-           jalr x0, r31       // Jump to the patch
+h1000_0000: auipc r31, disp_from_8940_to_1000_0040
```

Comment:
This 'auipc' operation here would have to be constructed from two instructions in practice, so a patch size of 16 bytes would have to be employed. Perhaps it would be better to modify the address of the replacement code to '1000_0940' to keep the explanation simple but consistent?